sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GatewayAddressCache.java

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
Please consider sorting the imports here as one of them may now be out of order.

Reply:
Done